test/fixtures/schema/object-fixed.json

```diff
@@ -143,7 +143,7 @@
                               "attributes": {
                                 "contentType": "application/schema+json"
                               },
-                              "content": "{\n  \"$schema\": \"http://json-schema.org/draft-04/schema#\",\n  \"type\": \"object\",\n  \"properties\": {\n    \"person\": {\n      \"type\": \"object\",\n      \"properties\": {\n        \"first_name\": {\n          \"type\": \"string\"\n        },\n        \"last_name\": {\n          \"type\": \"string\"\n        }\n      },\n      \"required\": [\n        \"first_name\",\n        \"last_name\"\n      ],\n      \"additionalProperties\": false\n    }\n  },\n  \"required\": [\n    \"person\"\n  ]\n}"
+                              "content": "{\n  \"$schema\": \"http://json-schema.org/draft-04/schema#\",\n  \"type\": \"object\",\n  \"properties\": {\n    \"person\": {\n      \"type\": \"object\",\n      \"properties\": {\n        \"first_name\": {\n          \"type\": \"string\",\n          \"enum\": [\n            \"\"\n          ]\n        },\n        \"last_name\": {\n          \"type\": \"string\",\n          \"enum\": [\n            \"\"\n          ]\n        }\n      },\n      \"required\": [\n        \"first_name\",\n        \"last_name\"\n      ],\n      \"additionalProperties\": false\n    }\n  },\n  \"required\": [\n    \"person\"\n  ]\n}"
```

Comment:
This is the only change that concerns me a little. What should we do for fixed values where the user did not supply any value?

Before we was treating this as non-fixed silently. Now it would use the "default" value, i.e, empty string etc. Should we perhaps issue a warning? Is this acceptable/expected?


```apib
# Resource [/example]

## Action [GET]

+ Response 200 (application/json)

    + Attributes
        - person (object, fixed)
            - first_name
            - last_name
```